packages/web/README.md

```diff
@@ -180,6 +180,20 @@ Check your browser console to see if you are using any of the deprecated functio
 
 ![Deprecations in the Browser's console](https://github.com/lmc-eu/spirit-design-system/blob/main/static/deprecations-browser-console.png?raw=true)
 
+## Feature flags
+
+This package uses feature flags to enable or disable some functionality. You can enable or disable them by loading
+the `featureFlags` module with a configuration. You have to do this before loading any other Spirit Web SCSS file.
```

Comment:
Please update `featureFlags` if also changing the file name.

Also, I'd welcome some kind of emphasis of the file order that is crucial for the flags to work. At least:

```suggestion
the `featureFlags` module with a configuration. You have to do this **before** loading any other Spirit Web SCSS file.
```